openbb_terminal/settings_controller.py

```diff
@@ -467,7 +467,6 @@ def call_export(self, other_args: List[str]):
             description="Select folder where to export data",
         )
         parser.add_argument(
-            "-f",
             "--folder",
```

Comment:
If you remove the "-f" from here, you should replace the `-f` below in line `478` for `--folder`.

And this should be checked in the other scenarios

Reply:
good catch, thanks